purchase_request_to_procurement/views/purchase_request.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+<data>
+
+    <!-- Purchase Request -->
+
+    <record model="ir.ui.view" id="purchase_request_form_view">
+        <field name="name">purchase.request.form (in purchase_request_to_procurement)</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='picking_type_id']" position="after">
+                <field name="warehouse_id"/>
+                <field name="location_id"/>
+            </xpath>
+            <xpath expr="//field[@name='line_ids']/form//field[@name='procurement_id']" position="replace"/>
+            <xpath expr="//field[@name='line_ids']/form//notebook"
+                   position="inside">
+                <page name="Procurement Order"
+                      string="Procurement Order">
+                    <group name="procurement_infos">
+                        <field name="procurement_id" />
+                        <field name="procurement_state"/>
+                    </group>
+                </page>
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="purchase_request_tree_view">
+        <field name="name">purchase.request.tree (in purchase_request_to_procurement)</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_search"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='picking_type_id']" position="after">
+                <field name="warehouse_id"/>
+                <field name="location_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <!-- Purchase Request Line -->
+
+    <record id="purchase_request_line_tree_view" model="ir.ui.view">
+        <field name="name">purchase.request.line.tree</field>
+        <field name="model">purchase.request.line</field>
+        <field name="inherit_id"
+               ref="purchase_request.purchase_request_line_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='supplier_id']" position="after">
+                <field name="procurement_state"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_request_line_form_view" model="ir.ui.view">
+        <field name="name">purchase.request.line.form</field>
+        <field name="model">purchase.request.line</field>
+        <field name="inherit_id"
+               ref="purchase_request.purchase_request_line_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='procurement_id']" position="replace"/>
```

Comment:
The only possibility to occurs errors should be an anchor with a very important precision pointing xpath axes too. I don't think it can be the case but if it is then I propose to move the original procurement field into a notebook. I made this move to be compilent with the added behavior of "purchase order" notebook (into purchase_request_to_rfq). But if you want I can just set `procurement_state` after `procurement_id` without moving it. Just tell me which one you prefer :smile: